packages/react-core/src/components/Menu/MenuContainer.tsx

```diff
@@ -37,6 +37,8 @@ export interface MenuContainerProps {
   zIndex?: number;
   /** Additional properties to pass to the Popper */
   popperProps?: MenuPopperProps;
+  /** Flag indicating if scroll on focus of the first menu item should occur. */
+  preventScrollOnItemFocus?: boolean;
```

Comment:
nit. Same comment here about the name of this prop. Can we prefix it with `should`?